docs/usage.md

```diff
@@ -108,6 +108,8 @@ it should infer the parser based on the filename:
 osv-scanner --lockfile ':/path/to/my:projects/package-lock.json'
 ```
 
+OSV-Scanner also supports submoduled and vendored [C/C++ dependencies](#cc-scanning). There is no need to specify a lockfile, OSV-Scanner will automatically find them and return any corresponding vulnerabilities. 
```

Comment:
@oliverchang I put this in the "Specify lockfiles" section. Would you prefer it moved up closer to the list of lockfiles? Or is it ok here.